src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -534,7 +534,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
 
         const assetMetadata = await this.props.actions.loadAssetMetadata(this.props.project, asset);
-        await this.props.actions.updateProjectTagsFromFiles(this.props.project);
+        await this.props.actions.updateProjectTagsFromFiles(this.props.project, asset.name);
```

Comment:
Now takes 500-3000 milliseconds to perform on my laptop for one asset. Better than the 7000 milliseconds it took before. 

Could just call once when editor page mounts for all assets, not per asset on asset select.